\chapter{Introduction}
\label{chapter_introduction}
%% Start the actual chapter on a new page.
\newpage

\noindent

Once upon a time, there was the first living organism, the 
First Universal Common Ancestor (FUCA).
We do not know when it lived.

% 
%    ----- FUCA
%
% ---+---- (time)
%    t0

One unknown day, FUCA speciated, resulting in two species.
This event doubled the biodiversity on Earth.
The two species, which we will call species A and B
are sister species.
We do not know what caused the speciation.

%     +---- A
%  ---+
%     +---- B
%
% -+--+--- (time)
%  t0 t1

Both species A had their unknown histories: they speciated themselves,
and they and/or their descendants went extinct. 
Extinction is a common event. Let's assume A and/or its clade went
exctint and that species B created a sister species C. Species B
and C will give rise to all biodiversity. This ancestor of species B and C 
is called the Last Universal Common Ancestor and lived around [then].

            
%     +--------x   A
%  ---+     
%     |     +------- B
%     +-----+
%           +------- C
%
% -+--+-----+--+---- (time)
%  t0 t1    t2 t3

The biodiversity derived from LUCA is important to us humans, 
apart from
that is has created us. A review paper (Cardinale et al.) shows that 
biodiversity unsually improves ecosystem services that are 
positive for human well-being.

%
% Ecosystem function
%
%   |  _______
%   | /
%   |/
%   +--------------
%
%     Biological diversity
%

Speciation is the process that increases biological diversity 
and is therefore worth to study. We can study the mechanism ('what
causes a speciation event?') or we can study the patterns of many
of such events ('is speciation rate constant through time?')
%
% [picture of speciation machanism] | [picture of a trait that increases speciation rate]
%                                   | 
% Speciation mechanism:             | Speciation events though time:                           
% what caused this speciation       | what are the patterns of
% event to occur?                   | speciation events?
% In this case, a geographics       | In this case, a trait that was a key innovation
% barrier caused the initial        | gave rise to a higher speciation rate
% species to diverge                | 
%

The mechanism of a speciation event has many facets.
For more than half a century ago, it was hypothesized
that speciation is caused by geographical isolation (e.g. Mayr, 1942)
or due to ecological factors (e.g. Lack, 1947). [other proximate 
causes of speciation]

% [picture of geographical isolation] | [picture of ecological factors]
% [from Mayr's paper]                 | [the Darwin finches of course!]
%                                     |
% Speciation can happen due to        | Speciation can happen due to
% a geographic barrier                | ecological factors
%

Instead of looking at the mechanism behind each speciation event,
we can also look at patterns of speciation events through evolutionary time.
How often do speciation and extinction events take place?
Are speciation and extinctions rates constant or do they change?
What causes a change in speciation or extinction rate?
Is there an upper limit on the number of species?

There are two ways to look in evolutionary time:
using fossils and molecular phylgenies.

% [picture of El Graeco fossil] | [](800px-Hominini_lineage.svg.png), from https://en.wikipedia.org/wiki/File:Hominini_lineage.svg
%                               | 
%  El Greaco                    | HS     Pan
%                               |  \    /
%                               |   \  /
%                               |    \/
%                               |     \

A classic way to look back in evolutionary time is to use fossils,
which are dated by the rock layers they are found in.
Using fossils has its limitations. First, it is mostly species with hard body
parts that are suitable to fossilize. Of such species, an organisms is still 
only rarely preserved, of which only a fraction under ideal circumstances. Of 
these fossils, only a fraction is discovered.
One example of a famous fossil is 'El Graeco', 
which may be the oldest known hominin \cite{fuss2017potential}, where
homonins are the tribe (taxanomic group) we Homo sapiens share with the Panini.

% [picture of adaptive radiation phylogeny] | [picture of DDD phylogeny]
%                                           |
% An adaptive radiation.                    | The number of species through time.
%                                           | [use Etienne, DDD]

A modern way to look back in evolutionary time is to use
heritable characteristicts (DNA, RNA, protein) 
of extant species to infer phylogenies. 
The field of phylogenetics is the research discipline that
intends to infer the most accurate phylogenies possible, 
regarding topology, speciation and extinction times,
optionally adding morphological data and/or fossil data.
Phylogenetics is applied in many settings, among
others, species classification,
forensics, conservation ecology
and epidemiology \cite{lam2010use}.

% ![](edge_tree.png)                          | ![](Pristis-pristis_Simon-Fraser-University-1000x667.jpg),
%                                             | from http://www.edgeofexistence.org/species/largetooth-sawfish/
% The ED (evolutionary distinctiveness)       | 
% of species A is higher than that of species | The Largetooth Sawfish (Pristis pristis) is at number 1 of
% B or C, as more                             | the EDGE list, with an EDGE Score of 7.38, 
% evolutionary history will be lost when      | with an ED Score of 99.298. Score is calculated by
% that species goes extinct.                  | ![](edge_score.svg)

Accurate phylogenetic trees are important for many reasons.
Here I will elaborate of the four examples of \cit{heath2008taxon},
which are in conservation biology (Crandall  et  al.,  2000), 
the understanding of evolved behavior (Martins,  1996), 
forensics (Hillis & Huelsenbeck, 1994; Metzker et al., 2002) and human 
health \cite{bush1999positive}. 

%
% [bush_et_al_1999] MP tree 
%

The importance of an accurate phylogenetic tree applied to human
health is demonstrated in \cite{bush1999positive}. This study
investigated which loci of the H3 hemagglutinin surface protein
are under selection, by constrasting asynonymous and synonymous
mutation rates along the branches of a phylogeny. 
In a preliminary analysis by the authors, they noted that
most selection rates were either below or above the 
statistical threshold depending on the phylogeny.
This study contributed to the selection of recommended 
composition of influenza virus vaccines.

One illustrating example is from conservation ecology, where 
accurate trees are important to correctly estimate the 
evolutionary distinctiveness (ED) of a species.
The ED is the amount of evolutionary history lost when a species would go extinct.
Together with the GE score ('Globally Endangered', from 0: 'Least Concern' 
to 4: 'Critically Endangered') the ED is used to assign an EDGE score 
to a species, so that conservation can prioritize its efforts on.

% ![](phylip.gif) |
%                 |
% PHYLIP logo     |

Phylogenetics has taken a huge flight, due to the massively increased
computational power and techniques. Milestones are Felsensteyn's work
in 1980, PHYLIP, the first software package for phylogenetic 
analysis (and is still in development!).
The Metropolis-Hasting algorithm allowed for efficient MCMC
sampling, allowing Bayesian phylogenetics to thrive. Contemporay
Bayesian phylogenetics tools are BEAST, BEAST2, MrBayes and RevBayes.

%
% [picture of BEAST2]
%

A clear example of what phylogenetics can do nowadays,
is the Tree Of Life, which uses 
3,083 genomes of 2,596 amino-acid positions 
to create one big phylogeny of all (sequenced) life on Earth,
which took 3,840 computational hours on a modern supercomputer [Hug et al., 2016].

%
% [Tree Of Life]
%
% The Tree Of Life [Hug et al., 2016]
%

To create the Tree Of Life from a protein alignment, 
there have been many assumptions 
made, regarding the evolution of a protein sequence, 
the rate(s) at which this happens and the rate(s) at which 
a branching/speciation event takes place. For example,
the amino acids are assumed to follow the LG [Si Quang 
Le and Olivier Gascuel, An Improved General Amino Acid Replacement Matrix] 
which uses empirical transition rates.

Selecting which inference model to use is non-trivial.
On can use the 'measure is knowing' approach, or do a model comparison.

The 'measure is knowing' starts by doing an inference with the simplest model.
After this, use an inference model slight more complex and see if the results
of the inference differ. If not, use the simpler method. Else, repeat with
a more complex model.

The model comparison approach uses algorithms to select the model that is
most likely to have generated the data. 

Again, having a phylogenetic tree being correct is important,
as decisions are made upon them. 

The effect of using a different inference model on the same data
is not investigated systematically. For example, 
if we use multiple deep calibration nodes, picking a
wrong clock model has no impact anymore \cite{duchene2014impact}.
The effect of tree prior remains uninvestigated \cite{alfaro2006posterior}.

%
% [part of LG transition rate matrix]
%
% Part of an LG transition rate matrix

On the topic of speciation, the most relevant part of the phylogenetic
assumptions is the tree prior. A tree prior is the Bayesian
form of the knowledge about the speciation process, at the
macro-evolutionary level. The tree prior determines how likely
new branches form (a speciation event) and how likely branches end (an
extinction event).

Picking the optimal tree prior is important, as this will result (by 
definition) in the most correct phylogeny to base decisions upon.

Due to the importance of a correct phylogeny, theoreticians
measure how well we can infer it. I will show some examples here:

%
%
% revell2005under.gif
%
% Figure from \cite{revell2005under}. At the left was the true tree.
% In the middle the inferred tree, that used the generative model
% At the left the inferred tree when using a too simple inference model 

We know, for example, that using a too simple inference model
on trees with a known gamma \cite{pybus2000testing}, the inference will bias
towards a negative gamma \cite{revell2005under}. A tree with
a negative gamma has a decreasing speciation rate. 
The biological relevance of this, is that an empiricist might
attribute the finding of a negative gamma to some biological
process, where in reality the negative gamma may be largely
attributed to his/her choice of using an under-parameterized inference
model.

%
% sarver2019choice.jpg
%
% Figure 3: Birth-death simulations.
%
% The top row of plots (A–D) refers to the 100-taxa cases, whereas the bottom row (E–H) refers to the 25-taxa cases. The median estimates of λ or r, estimated from the 10 original trees, are used as data for each boxplot. The title of each subplot refers to the simulation conditions. Each combination of tree priors and molecular clocks under which trees are estimated is listed on the x-axis. The distribution of estimates from the original trees is also displayed. Parameter estimates are highly congruent with the original trees under each set of simulation conditions.

Another example of a study that measures the inference error
we make when using an incorrect tree prior,
shows that this has little effect on the correctness of the parameter 
estimation \cite{sarver2019choice}. Note that this study already conflicts
with the formerly discussed study (\cite{revell2005under})!

The biological research shown in this thesis also investigates
the inference error made from using an incorrect tree prior.
Novel about the research in this thesis is the use of non-standard tree
priors and the contribution of a framework to easily investigate
the inference error.

Because speciation models and tree priors are pivotal in this thesis, 
I will here describe the two standard ones, as well as the two
novel non-standard ones:

The most basic speciation model
is the Yule model [Yule, 19..] which assumes that speciation
is constant and there is no extinction.
[Research on fossils with Yule model would be fun].
The Yule model predict that the number of extant species
grows exponentially through time.

%
% [example Yule tree]  | [LTT plot for Yule model]
%                    |
% An example Yule tree | A lineages-through-time plot of the Yule model.
%                    | 

The Birth-Death model [Nee et al., 1994] is an extension of the
Yule that allows for a constant extinction rate. 
If the speciation rate exceeds the extinction rate,
also the BD model predicts that the number of extant species
grows exponentially through time. If the extinction rate exceeds
the speciation rate, the number of lineages is expected to decline
exponentially. The latter is biologically irrelevant.

%
% [example BD tree]  | [LTT plot for BD model]
%                    |
% An example BD tree | A lineages-through-time plot of the BD model.
%                    | 

It is clear that an exponential growth in the expected number of lineages
is biologically nonsense. 
To state the obvious: a finite area (Earth) results in a finite number of species. 
Applying the BD model to molecular data already shows that it does not
always hold, see figure [below]

%
% n_lineages
%
%   |      _______________
%   |   __/
%   | _/
%   |/
%   +---------------------
%
%       time
%
% An LTT plot for bird/lizards from [Etienne and Haegeman, 2012]
%

One extension of the BD model that offers a biological explanation is the
diversity-dependent model [Etienne and Haegeman, ?2012] in which the
speciation rate decreases up until a certain carrying capacity is reached.
This carrying capacity amounts to the number of niches in an environment.

%
% n_lineages
%
%   |      _______________
%   |   __/
%   | _/
%   |/
%   +---------------------
%
%       time
%
% An LTT plot from molecular phylogeny used by [?Lovelocke]
%

The are multiple other extensions of the BD model, filling in different
biological aspects that are (purposefully) lacking. The time-dependent BD 
model [?Lovelocke], for example, assumes that speciation and extinction 
rates are time-dependent. The explanation for this would be that speciation
is mostly driven by abiotic factors, like for example, temperature,
which is known to fluctuate.

%                           |
% temperatue                | speciation rate
%                           |   
%   |   _     _             |  |   __      __
%   |  / \   / \            |  |  /  \    /  \
%   | /   \_/   \           |  | /    \__/    \___
%   +-----------------      |  +--------------------
%             time          |           time
%                           |
% Temperature through time  | Speciation rate through time
% from [?Lovelocke]         | from [?Lovelocke]

When constructing a phylogeny from Lake Tangyanika [or some
other adaptive radiation place], we can find evidence that
one clade has a sudden higher speciation rate than another.
One explanation for this, is to assume that a certain key
innovation (e.g. jaw/opsin) causes a change in speciation
rate. This idea is incorporated by the trait-dependent 
BD model [Maddison].

%
%  +-------------
% ++
%  |       +----
%  |   +---+
%  |   |   +----   
%  +-X-+  
%      | +------
%      +-+
%        +------
%
% A phylogeny used by Maddison. X denotes a key innovation
%

Another facet of speciation is that speciation takes time. It
takes time for the build-up of reproductive isolations and it
takes time for us humans to recognize the two-not-one species.
A biological example is from [Fennesy, 201?] in which
some new giraffe species have been discovered by sequencing
part of their DNA. Although these new species have been
'discovered' recently, the had been no gene flow between species
for already two million years.

%
% [phylogeny from Fennesy]
%
% Phylogeny from Fennesy
%

Using the BD model in species that are slow to speciate, will cause
an underestimation of the number of lineages in the present (as in the
giraffes), in effect possibly giving the illusion that speciation 
slows down, where in reality it does not. Note that also the
time-dependent and diversity-dependent speciation models also
offer an explanation of this knowledge.

%
% n_lineages
%
%   |      _______________
%   |   __/
%   | _/
%   |/
%   +---------------------
%
%       time
%
% An LTT plot from molecular phylogeny used by [Etienne and Rosindell]
%

The speciation model that encorporates the fact that speciation takes
time is the PBD model [Etienne and Rosindell, 201?]. This extension of the
BD model adds that each species has a state: a 'good' species is
a species that is recognized as such, where an 'incipient' species is
not yet recognized yet. It takes time for an incipient species to
become a good species. This time is called the speciation time [or those
other times Rampal uses].

%
% [PBD phylogeny from Etienne and Rosindell]
%
% PBD phylogeny from Etienne and Rosindell
%

Another facet of speciation uncaptured by the BD model is the
effect of a geographical isolation. When a habitat (lake or mountain range)
gets separated, this will have an effect on both isolated communities.
One can imagine that this triggers a speciation event in multiple species
of both communities at the same time. This is posed as one alternative
explanation for the high speciation rate in lake Tangyanika, where the water 
level rises and falls with ice ages, triggering co-occurring speciation
events each change. 

%
% [phylogeny from Lake Tangyanika, use reference from Giovanni's MBD article]
%
% Phylogeny from Lake Tangyanika
%

Where the BD model allows for exactly one speciation model at one timepoint,
the Multiple-Birth Death (MBD) model allows for co-occurring 
speciation events [Laudanno, 201?]. For systems that are likely
to have such co-occurence, we can improve our inference.







[WHAT DO PEOPLE WANT TO KNOW]
